src/utils/theme.js

```diff
@@ -18,19 +18,22 @@ const DEFAULT_COLORS = {
 };
 
 function getThemeColor(name) {
-  if (configs.APP_CONFIG && configs.APP_CONFIG.theme && configs.APP_CONFIG.theme[name])
-    return configs.APP_CONFIG.theme[name];
+  const themeId = window.APP?.store?.state?.preferences?.theme;
+
+  const theme =
+    themeId &&
+    configs.APP_CONFIG &&
+    configs.APP_CONFIG.theme &&
+    configs.APP_CONFIG.theme.themes &&
+    configs.APP_CONFIG.theme.themes.find(theme => theme.id === themeId);
```

Comment:
```suggestion
    configs?.APP_CONFIG?.theme?.themes?.find(theme => theme.id === themeId);
```

then remove lines 25 -> 27 definitely my favorite recent JS upgrade :D